nomenclature/definitions/variable/technology/technologies.yaml

```diff
@@ -538,103 +538,83 @@ Investment|Energy Supply|Liquids|Oil:
    
    # description of residential demand flexibility potentials for 10 household technologies - potential curtailable load
    
-Load Shifting Potential|Electricity|Residential|Refrigeration:
-   description: Maximum curtailable load in a given region/time frame,
-                due to changing household freezer/refrigerator usage.
+Maximum Curtailment|Electricity|Load Shifting|Residential|Refrigeration:
```

Comment:
Curtailement should not be used as it means something different (see global comment)

Reply:
"Maximum Reduction"?

Reply:
Maximum reduction could do it but maybe we would need maximum reduction of sth?
Why not capacity? (load shifting is already in the variable name)?